apps/integration/src/app/integrations/flyout-tabs/flyout-tabs.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component } from '@angular/core';
+import {
+  SkyFlyoutConfig,
+  SkyFlyoutInstance,
+  SkyFlyoutService,
+} from '@skyux/flyout';
+
+import { FlyoutTabsContentComponent } from './flyout-tabs-content.component';
+
+screenY;
+@Component({
+  selector: 'app-flyout-demo',
```

Comment:
Nitpicky - would you mind changing this selector to `app-flyout-tabs` so that it matches the file name and the component name?